plugins/modules/remote_management/redfish/redfish_command.py

```diff
@@ -631,7 +641,8 @@ def main():
                     transfer_protocol_type=dict(),
                     transfer_method=dict(),
                 )
-            )
+            ),
+            strip_etag_quotes=dict(type='bool', default=False)
```

Comment:
That will make the diff look cleaner the next time a new option is added:
```suggestion
            strip_etag_quotes=dict(type='bool', default=False),
```